### PR TITLE
Homework2

### PR DESCRIPTION
**Pull request by KseniaKotova**
Task: https://www.figma.com/file/16bdlWFoZXXIpyyZmwy9JU/Logoipsum?node-id=0%3A1
Created:  
image

### CODE REVIEW

File: lesson 2/style.css [hunk 1]
```diff
@@ -0,0 +1,122 @@
+body{
+    margin: 0 auto;
+    width: fit-content;
+    overflow-x: hidden;
+    font-family: 'Mulish', sans-serif;
+    background-color: #ffffff;
+}
+
+/* h1, h2, h3, h4, h5, h6{
+    margin: 0;
```
Comment (shuliakleonid) [inline lesson 2/style.css]: if you do not need comments, delete it

File: lesson 2/style.css [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+}
+
+.header__nav{
+    margin-top: 40px;
+    display: block;
+}
+.nav_link{
+    display: inline-block;
+    
+    padding-top: 14px;
```
Comment (shuliakleonid) [inline lesson 2/style.css]: empty row

File: lesson 2/style.css [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+button{
+    cursor: pointer;
+}
+.header__btn{
+   padding-top: 30px;
+   margin-left: 120px;
+   margin-right: 100px;
+}
+.btn_inner{
```
Comment (shuliakleonid) [inline lesson 2/style.css]: after class do please empty line

File: lesson 2/style.css [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+.content__hero{
+    padding-top: 100px;
+    margin-left: 184px;
+}
+
+.content{
+    position: absolute;
+    margin-left: -60px;
+    
+
```
Comment (shuliakleonid) [inline lesson 2/style.css]: empty row

**Review by shuliakleonid — CHANGES_REQUESTED** [review]

File: lesson 2/index.html [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+                <div class = "hero__sub">
+                <h1 class = "hero__title">Grow your subscription business</h1>
+                <p class = "hero__text">Outcome-centered products that reduce churn, optimize pricing, and grow your subscription business end-to-end.</p>
+                <div class = "hero__button">
+                    <button class = "hero__btn">Get Started</button>
+                </div>
+            </div>
+            <div class = "hero__logo">
+                <img src = "pose_7.png" width="610" height="644">
+            </div>
```
Comment (shuliakleonid) [inline lesson 2/index.html]: add alt

File: lesson 2/style.css [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+    text-decoration:underline;
+}
+
+button{
+    cursor: pointer;
+}
+.header__btn{
+   padding-top: 30px;
+   margin-left: 120px;
+   margin-right: 100px;
```
Comment (shuliakleonid) [inline lesson 2/style.css]: can you write with main margin

**Review by shuliakleonid — CHANGES_REQUESTED** [review]

**Comment by shuliakleonid** [comment]
# Good work

File: lesson 2/index.html [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+    <link href="https://fonts.googleapis.com/css2?family=DM+Sans:wght@700&family=Mulish&display=swap" rel="stylesheet">
+    <title>Logoipsum</title>
+</head>
+<body>
+    <header class = "header">
+        <div class = "container">
+            <div class = "header__inner">
+                <div class = "header__logo"><img src="Logo Color.png" alt = "logo"></div>
+                <nav class = "header__nav">
+                    <a class = "nav__link" href = "#">Home</a>
```
Comment (shuliakleonid) [inline lesson 2/index.html:19]: <nav class = "header__nav">
your may write like <nav class = "nav"> it's correct or if you need two class 
<nav class = "header__nav nav">
but it's not mistake

File: lesson 2/index.html [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+                    <a class = "nav__link" href = "#">Features</a>
+                    <a class = "nav__link" href = "#">Pricing</a>
+                    <a class = "nav__link" href = "#">Blog</a>
+                </nav>
+                <div class = "header__btn">
+                    <button class = "btn__inner">Get Started</button>
```
Comment (shuliakleonid) [inline lesson 2/index.html:25]: the same with button

File: lesson 2/index.html [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+                </div>
+            </div>
+            <div class = "hero__logo">
+                <img src = "Hero 3D.png" alt = "hero" width="610" height="644">
+            </div>
+        </div>
+    </div>
+    </section>
+    <div class="logos">
+        <div class="logo__about">
```
Comment (shuliakleonid) [inline lesson 2/index.html:48]: may be logo

**Review by shuliakleonid — APPROVED** [review]

**Review by AlexisKozlov — APPROVED** [review]
Thanks, great work!

**Review by egkirpichev — APPROVED** [review]
👍

**Pull request merged**